surveys/surveys.go

```diff
@@ -2,11 +2,12 @@ package surveys
 
 import (
 	"encoding/json"
-	"github.com/AreaHQ/jsonhal"
-	"github.com/ONSdigital/go-launch-a-survey/settings"
 	"log"
 	"net/http"
 	"regexp"
+
+	"github.com/AreaHQ/jsonhal"
+	"github.com/ONSdigital/go-launch-a-survey/settings"
```

Comment:
why have these moved?

Reply:
@nealedj @shahi645 Auto lint by VS code. Its doing it for a reason I guess, modules first.
Nothing to be concerned about.